common/browser.go

```diff
@@ -409,6 +409,11 @@ func (b *Browser) Close() {
 	b.conn.IgnoreIOErrors()
 	b.browserProc.GracefulClose()
 
+	if b.browserProc.Killed(b.ctx) {
```

Comment:
This i think is racey. What if the kill process takes a bit longer, which would mean that we can still block on the cdp write/read command.